e2e-tests/src/test/java/com/dnastack/ga4gh/dataconnect/adapter/DataConnectE2eTest.java

```diff
@@ -161,7 +161,7 @@ public class DataConnectE2eTest extends BaseE2eTest {
     private static final String INDEXING_SERVICE_URI = optionalEnv("E2E_INS_BASE_URI", "http://localhost:8094");
     private static final String INDEXING_SERVICE_RESOURCE_URI = optionalEnv("E2E_INS_RESOURCE_URI", "http://localhost:8094/");
 
-    private static final boolean globalMethodSecurityEnabled = Boolean.parseBoolean(optionalEnv("E2E_GLOBAL_METHOD_SECURITY_ENABLED", "false"));
+    private static final boolean globalMethodSecurityEnabled = Boolean.parseBoolean(optionalEnv("E2E_GLOBAL_METHOD_SECURITY_ENABLED", "true"));
```

Comment:
Please check the README to ensure the instructions for running E2E tests still work for those who use this open source project with OAuth 2 resource server checks turned off.

Reply:
@prajjwolmondal is this change even intentional?